pkg/virt-launcher/virtwrap/BUILD.bazel

```diff
@@ -80,6 +81,7 @@ go_test(
     data = glob(["testdata/**"]),
     embed = [":go_default_library"],
     embedsrcs = ["testdata/migration_domain.xml"],
+    tags = ["cov"],
```

Comment:
Can you please explain this line?

Reply:
Bazel coverage is required to run the regular `go_test` instead of `ginkgo_test` ergo `nocov` and `cov` tags were introduced to avoid running both `go_test` and `ginkgo_test`.

The target called is based on what's defined in the bazelrc: https://github.com/kubevirt/kubevirt/blob/main/.bazelrc#L34-L36